src/js/jsx/sections/export/ExportPanel.jsx

```diff
@@ -256,7 +252,10 @@ define(function (require, exports, module) {
                         </div>
                     </TitleHeader>
                     <div className={containerClasses}>
-                        {containerContents}
+                        <ExportList {...this.props}
```

Comment:
Yeah that shouldn't have happened, weird. What needs to happen is actually when containerContents is defined in lines 184:188, the <div> around <ExportList> should be taken out so that the <ExportList> width is the width of the full div. Changes being made now.